superset/assets/src/dashboard/components/Dashboard.jsx

```diff
@@ -215,6 +215,29 @@ class Dashboard extends React.PureComponent {
     });
   }
 
+  isFilterkeyExistInLinkedSlices(chart, filterKey) {
+    const linked_slices = chart.formData.linked_slice;
+    const propExist = chart.formData.hasOwnProperty("linked_slice");
+    if (linked_slices instanceof Array) {
+      return propExist && linked_slices.indexOf(parseInt(filterKey)) != -1
+    }
+    return propExist && linked_slices === parseInt(filterKey)
+  }
+
+  getLinkedSlicesFilters(chart) {
+    const filters = {};
+    const linked_slices = chart.formData.linked_slice;
```

Comment:
Wrap this function under propExists also handle the empty filter object in usage